src/content/en/updates/2018/07/page-lifecycle-api.md

```diff
@@ -616,7 +616,7 @@ The above code does three things:
 <aside class="warning">
   <strong>Warning!</strong>
   This code yields different results in different browsers, as the order
-  (and reliability) of events has not been consistently implemented. To learn
+  and reliability of events has not been consistently implemented. To learn
```

Comment:
This small change makes it a bit more clear that reliability of the events across browsers is problematic.